resources/responses.js

```diff
@@ -203,6 +206,15 @@ let Responses = {
     };
     return obj;
   },
+  bigAoe: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Extreme raid damage!',
```

Comment:
how about just ... big aoe, to match other triggers?

Reply:
changed it :)